README.md

```diff
@@ -53,11 +54,52 @@ Run the following commands to prepare the environment and management cluster:
 
 Apply customizations provided by this repo:
 
-**_Optional_**: This step is needed if your mangement cluster is an OpenShift Cluster
+we provide GitOps ways to apply customizations
+
+1. Bootstrap AgoCD instance on management Cluster
+
+**_Note_**: These steps will disable the default argocd instance with setting DISABLE_DEFAULT_ARGOCD_INSTANCE as 'true' and will provision a argocd instance in namespace named as  "gitops". 
```

Comment:
Please explain why this is needed so the consumer of the content understand the reasoning behind this.  ... and is there really a good reason to do this? Unless there's technical reasons why this is really needed, it just seems like extra - and perhaps unnecessary - overhead to maintain.

Reply:
The default openshift-gitops instance is managed by CRD GitopsService, which does not allow consumer to customize their argcd instance like "resourceTrackingMethod: annotation " and  add resourceHealthChecks for integrations like crossplane resources.  That's the reason I disable the default argoCD instance and use a customized one.